pnpm-lock.yaml

```diff
@@ -11085,7 +11085,7 @@ snapshots:
 
   dmg-builder@25.1.8(electron-builder-squirrel-windows@25.1.8):
     dependencies:
-      app-builder-lib: 25.1.8(dmg-builder@25.1.8(electron-builder-squirrel-windows@25.1.8))(electron-builder-squirrel-windows@25.1.8(dmg-builder@25.1.8))
+      app-builder-lib: 25.1.8(dmg-builder@25.1.8)(electron-builder-squirrel-windows@25.1.8)
```

Comment:
Beautiful, I didn't even notice this at first, but I remember that some of the minor v9 pnpm release fixed the problem with recursive peer deps.